app/src/main/java/com/example/thriftbooks/PostsAdapter.java

```diff
@@ -57,10 +66,12 @@ class ViewHolder extends RecyclerView.ViewHolder {
         private final ImageView displayPicture;
         private final TextView tvBookAuthor, timeAgo;
         private final TextView tvBookTitle;
-        private final TextView tvBookType;
+        private final TextView tvBookType, tvMoreAboutPost;
         private final TextView tvBookCondition;
         private final ImageButton ibComment;
-        private final ImageButton ibMessage; //Gonna use ibMessage later for messaging the book owner
+        private final Button btnSend;
+        private final EditText etSendMessage;
+        //private final ImageButton ibMessage; //Gonna use ibMessage later for messaging the book owner
```

Comment:
Thanks for adding the comment '//Gonna use ibMessage later for messaging the book owner'. This really helps when reviewing because otherwise it would look like just dead code to me